### ISSUE TITLE
Study changes to t/op/*.t (11 of 12)

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#221]
In this ticket we will study our progress in pursuit of [Objective 2, "strict-by-default"](https://github.com/atoomic/perl/issues/153), for the following files in the Perl core distribution:

```
t/op/threads.t
t/op/tiearray.t
t/op/tiehandle.t
t/op/time.t
t/op/time_loop.t
t/op/tr.t
t/op/universal.t
t/op/unshift.t
t/op/vec.t
t/op/ver.t
t/op/warn.t
t/op/while.t
t/op/write.t
```
These files should be evaluated in light of the guidelines presented on [this wiki page](https://github.com/atoomic/perl/wiki/How-to-Start-Working-with-Tests-in-the-Perl-Core-Distribution).  Please post the summary of your findings in a single comment in this issue.

**First-time contributors:**  (1) Please post a comment to this ticket indicating you are working on it.  That will enable us to assign the ticket to you.  (GitHub won't let us do that until it already thinks you're part of the project.)  (2) You should subscribe to this repository so that you get email updates.

Thank you for helping to improve the Perl programming language.

**Comment by brainbuz** [comment]
I can take a few more of these.

**Comment by brainbuz** [comment]
file | pass | strictures removed | reviewed diffs | notes
-- | -- | -- | -- | --
t/op/threads.t | x | --- | x |  
t/op/tiearray.t | x | x | x |  
t/op/tiehandle.t | x | --- | x |  
t/op/time.t | x | x | x |  
t/op/time_loop.t | x | x | x |  
t/op/tr.t | x | x | x |  
t/op/universal.t | x | x | x |  
t/op/unshift.t | x | x | x |  
t/op/vec.t | x | x | x |  
t/op/ver.t | x | x | x |  
t/op/warn.t | x | x | x |  
t/op/while.t | x | x | x |  
t/op/write.t | x | --- | x | Line 324 strict appears in a string passed to format.
  |   |   |   | Removing at 1649
  |   |   |   | Removing at 1662
  |   |   |   | another format at 1920

write.t is testing write and format functions using blocks that look like perl code, my pull request will eliminate 2 of the stricts from these and leave the other 2.

**Comment by jkeenan** [comment]
P.r. merged into `alpha` in commit c5c5fb4fb2.  Review looks complete. Closing ticket.

Thank you very much.
Jim Keenan
